roaringbitmap.c

```diff
@@ -2053,3 +2053,30 @@ rb_cardinality_final(PG_FUNCTION_ARGS) {
         PG_RETURN_INT64(card1);
     }
 }
+
+//bitmap run optimize
+PG_FUNCTION_INFO_V1(rb_or);
+Datum rb_or(rb_runoptimize);
+
+Datum
+rb_or(PG_FUNCTION_ARGS) {
+    bytea *serializedbytes = PG_GETARG_BYTEA_P(0);
+    roaring_bitmap_t *r;
+    bool optimized;
+
+    r = roaring_bitmap_portable_deserialize(VARDATA(serializedbytesin));
```

Comment:
Should you be doing any null checks before running this? Like functions above?

Reply:
I don't think that's necessary. functions like rb_to_array seem to be able to avoid it entirely.